rest_framework_swagger/views.py

```diff
@@ -109,6 +116,13 @@ def get(self, request):
             }),
         })
 
+    def get_base_path(self):
+        protocol = SWAGGER_SETTINGS.get('protocol', 'http')
```

Comment:
This is also inconsistent with the provided docs. The one case where I was foggily under the impression this would be beneficial is when is_secure is false but you do want https.